### PR TITLE
Fix file upload by chunks method to use the right sequence of parameters

### PR DESCRIPTION
**Pull request by WMKhant**

### CODE REVIEW

**Review by bhavin07 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.1.1
+- Fix file upload chunk.
```
Comment (myfoxtail) [inline CHANGELOG.md]: @WMKhant we need a better description of the change here. I suggest "Fix file upload by chunks method to use the right sequence of parameters"

Comment (myfoxtail) [reply CHANGELOG.md]: Also, please update the name of the PR accordingly.

**Review by myfoxtail — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (WMKhant) [reply CHANGELOG.md]: @myfoxtail update description

Comment (myfoxtail) [reply CHANGELOG.md]: Thank you @WMKhant .

**Pull request merged**